web/components/tipper.tsx

```diff
@@ -13,25 +12,28 @@ import { Row } from './layout/row'
 import { LIKE_TIP_AMOUNT } from 'common/like'
 import { formatMoney } from 'common/util/format'
 
-export function Tipper(prop: { comment: Comment; tips: CommentTips }) {
-  const { comment, tips } = prop
+export function Tipper(prop: {
+  comment: Comment
+  myTip: number
+  totalTip: number
+}) {
+  const { comment, myTip, totalTip } = prop
 
   const me = useUser()
-  const myId = me?.id ?? ''
-  const savedTip = tips[myId] ?? 0
+  const savedTip = myTip
```

Comment:
Why have both `savedTip` and `myTip`?

Reply:
I have no idea, this component code looks ridiculous at a glance but I don't feel like trying to deal with it right now so I just preserved existing logic.

Reply:
You could delete this line and just refer to `myTip`

Reply:
I grudgingly admit that I can at least do that.